app/ui/browser-modern/views/browser/tabbar/tab-pointer-area.jsx

```diff
@@ -57,4 +60,12 @@ TabPointerArea.propTypes = {
   onMouseUp: PropTypes.func,
 };
 
-export default TabPointerArea;
+function mapStateToProps(state, ownProps) {
+  const page = PagesSelectors.getPageById(state, ownProps.pageId);
+
+  return {
+    tooltipText: page ? page.title || page.meta.title || page.location : '',
```

Comment:
This looks like almost the same order as tab-content's tabTitle -- should probably be consistent, maybe moved into utility?